pkg/ansible/generate.go

```diff
@@ -0,0 +1,257 @@
+package ansible
+
+import (
+	"bytes"
+	"fmt"
+	"text/template"
+
+	coapi "github.com/openshift/cluster-operator/pkg/apis/clusteroperator/v1alpha1"
+)
+
+const invTemplate = `
+[OSEv3:children]
+masters
+nodes
+etcd
+
+[OSEv3:vars]
+
+[masters]
+
+[etcd]
+
+[nodes]
+`
+
+// varsTemplate defines all the data we consider unchangable for all clusters.
```

Comment:
I find this comment confusing. The comment first talks about defining all the unchangeable data. Then the comment talks about how the data is changed.

Also, if we keep this comment, change `unchangable` to `unchangeable`.

Reply:
Will update it.